base/experimental.jl

```diff
@@ -152,6 +166,12 @@ macro compiler_options(args...)
                 a = a === false || a === :no  ? 0 :
                     a === true  || a === :yes ? 1 : error("invalid argument to \"infer\" option")
                 push!(opts.args, Expr(:meta, :infer, a))
+            elseif ex.args[1] === :max_methods
+                a = ex.args[2]
+                a = a === :default ? 3 :
+                  a isa Int ? ((0 < a < 5) ? a : error("\"max_methods\" must be between 0 and 5.")) :
```

Comment:
I'm not sure if "between" always indicate inclusive range?

Reply:
I've made the message more clear.